data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
+  "origin": {
+    "date_human-readable": "mid-4th c. CE",
+    "date_normalized": "0325/0375",
+    "language": "Greek",
```

Comment:
controlled vocabulary. This does raise the question of if we will have different work authorities for the various translations of a text? (e.g., will we have one for the Greek original, Syriac translations, Arabic translations, etc.) Will we have 'translation_of' relationships set between these work records? Likely a discussion for the ad board, but my instinct is that we should have separate records for the versions of ancient texts, especially since we have multiple languages represented in the mss and presumably could have Greek, Arabic, and Syriac versions of Basil's Hexaemeron. The question here would be how to model relationships where tranmission direction isn't clear (i.e., when the translation from and into directions aren't clear-cut). At the same time, this distinction between translations may break down distinctions between work and expression, etc. I'm not sure I want to introduce a whole other object type though (e.g., expression, which somehow in between work and textual artifact?). Definitely need to reflect and discuss further.